README.md

```diff
@@ -79,8 +79,15 @@ with the next versions released since this project's gem will not be present in
 your Gemfile.
 
 ## Development
+
 For information about contributing to this project please refer to this
-[document](docs/development.md).
+[document](docs/development.md). There you'll find information on tasks like:
+
+* Testing the extension
+* Running the sandbox
+* Updating the changelog
+* Releasing new versions
+* Docker development
```

Comment:
Provided more details on the Development documentation so that developers will know where to find more information on tasks such as how to run the sandbox. Addresses https://github.com/nebulab/solidus_starter_frontend/pull/171#issuecomment-884715051.